src/pls.jl

```diff
@@ -217,10 +225,35 @@ function fitted!(v::AbstractArray{T}, m::LinearMixedModel{T}) where T
     v
 end
 
+fitted!(v::AbstractArray{T}, m::LinearMixedModel{T}) where {T} = fitted!(v, m, m.trms)
+
 StatsBase.fitted(m::LinearMixedModel{T}) where {T} = fitted!(Vector{T}(nobs(m)), m)
 
 StatsBase.predict(m::LinearMixedModel) = fitted(m)
 
+function StatsBase.predict(m::LinearMixedModel{T}, test::AbstractDataFrame; #, train::AbstractDataFrame;
+	 contrasts=Dict()) where T
+    mf = ModelFrame(m.formula, test, contrasts=contrasts)
+    X = ModelMatrix(mf).m
+    n = size(X, 1)
+    T = eltype(X)
```

Comment:
`T` is already defined in the function signature.  Do you really want to override that here?

Reply:
Oh, no.